modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with the condition type, informing you of the current state of your operator's health. The following definitions cover a list of some common ClusterOperator condition types. Some operators may have additional condition types with operator-specific language, which are not included here. 
```

Comment:
[capitalizing "Operator"](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator) and some minor suggestions for removing past tense and negative constructions:
```suggestion
The status of cluster Operators includes their condition type, informing you of the current state of your Operator's health. The following definitions cover some common ClusterOperator condition types. Operators that have additional condition types and use Operator-specific language have been omitted.
```